lib/std/json/fmt.zig

```diff
@@ -0,0 +1,46 @@
+const std = @import("std");
+
+const stringify = @import("stringify.zig").stringify;
+const StringifyOptions = @import("stringify.zig").StringifyOptions;
+
+/// Returns a formatter that formats the given value using stringify.
+pub fn fmt(value: anytype, options: StringifyOptions) Formatter(@TypeOf(value)) {
+    return Formatter(@TypeOf(value)){ .value = value, .options = options };
+}
+
+/// Formats the given value using stringify.
+pub fn Formatter(comptime T: type) type {
+    return struct {
+        value: T,
+        options: StringifyOptions,
+
+        pub fn format(
+            self: @This(),
+            comptime fmt_spec: []const u8,
+            options: std.fmt.FormatOptions,
+            writer: anytype,
+        ) !void {
+            _ = fmt_spec;
+            _ = options;
+            try stringify(self.value, self.options, writer);
+        }
+    };
+}
+
+test "json fmt" {
```

Comment:
```suggestion
test {
```
Seems a bit pointless to give this test a name?

Reply:
I gave it a name so I could call it out with a test-filter.

Reply:
Sounds like to me there could be made an improvement to the test filter so that you can still filter a test even if it doesn't have a name. It should just use the namespace name and probably also include the path kinda. So the "json/fmt.zig" part. Something like that maybe.

Reply:
This test is a pretty good example of what the `fmt` function does. What about:
```suggestion
test fmt {
```
so that it shows up in the docs for `fmt`? can you still find it with test filters when you use this kind of syntax?

Reply:
Yeah, unnamed tests should never really be used for actual testing - named tests exist for a reason, they make the tests more readable, filterable, etc - but this definitely seems like a perfect example of a decl test.